influxdb3_write/src/last_cache.rs

```diff
@@ -310,21 +319,35 @@ pub(crate) struct LastCache {
     /// the [`remove_expired`][LastCache::remove_expired] method.
     ttl: Duration,
     /// The key columns for this cache
-    key_columns: Vec<String>,
-    /// The Influx Schema for the table that this cache is associated with
-    schema: Schema,
+    key_columns: IndexSet<String>,
```

Comment:
Maybe add a note on why `IndexSet` was used, i.e., for fast lookup in addition to iteration to the doc comment.